src/pagerprinter/plugins/txt.py

```diff
@@ -0,0 +1,62 @@
+"""
+pyttsx (text to speech)
+Copyright 2013 Shane Rees <https://github.com/Shaggs/>
+
+This plug-in is designed to read out a copy of the received turnout 
+Page for those that maybe in the station or form those coming in 
+and need details.
+
+it reads only parts of the page. They are RESPOND (job type), Alarm Level,
+address, and any extra info after the ==
+
+the speed is hard coded at this stage.
+
+
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from __future__ import absolute_import
+from . import BasePlugin
+import pyttsx
+import re
+engine = pyttsx.init()
+rate = engine.getProperty('rate')
+engine.setProperty('rate',50)
+volume = engine.getProperty('volume')
+engine.setProperty('volume',100)
+class txt(BasePlugin):
```

Comment:
You should use class names that begin with an uppercase letter.

I wouldn't use the name `txt` either, as it is ambiguous to it's function.  You should probably use something like `TTSPlugin` instead.